src/controllers/scripting/legacy/controllerscriptinterfacelegacy.h

```diff
@@ -24,6 +24,11 @@ class ControllerScriptInterfaceLegacy : public QObject {
 
     Q_INVOKABLE double getValue(const QString& group, const QString& name);
     Q_INVOKABLE void setValue(const QString& group, const QString& name, double newValue);
+
+    Q_INVOKABLE QJSValue getRuntimeData();
+    Q_INVOKABLE void setRuntimeData(const QJSValue& value);
+    Q_INVOKABLE QJSValue onRuntimeDataUpdate(const QJSValue& callback);
```

Comment:
```suggestion
    Q_INVOKABLE QJSValue makeRuntimeDataConnection(const QJSValue& callback);
```
As far as I can see, this is very similar to `makeConnection` and should get a similar name for consistency across the API therefore.